audio/out/ao_pulse.c

```diff
@@ -350,6 +357,7 @@ static int pa_init_boilerplate(struct ao *ao)
         pa_threaded_mainloop_unlock(priv->mainloop);
     }
     uninit(ao);
+    pa_proplist_free(props);
```

Comment:
nit: reverse init order for clean-up would probably be good.

Since the prop list was probably done after AO init, then freeing it should be before the AO uninit happens.